src/main/java/com/mate/App.java

```diff
@@ -1,12 +1,71 @@
 package com.mate;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ForkJoinPool;
+import java.util.concurrent.ForkJoinTask;
+import java.util.concurrent.Future;
+import org.apache.commons.collections4.ListUtils;
 import org.apache.log4j.Logger;
 
 public class App {
```

Comment:
split this logic into 2 separate classes 
1 - will calculate list sum based on ExecutorService
2 - will do the same using ForkJoing pool 
and you won't need App class (test those 2 with unit tests)